lib/filter.ts

```diff
@@ -58,7 +58,7 @@ export class Filter {
 				process.cwd(),
 				file.gulp.base
 			) + '/';
-			if (file.gulp.path.substring(base.length) === searchFileName) {
+			if (path.resolve(base, searchFileName) === file.gulp.path) {
```

Comment:
Have you tested this on Windows? I'm afraid this might not work since Windows uses a backward slash instead of a forward slash.